### PR TITLE
docs: Add brief instructions on running a local dev server

### PR DESCRIPTION
**Pull request by matthewlloyd**
Signed-off-by: Matthew Lloyd <github@matthewlloyd.net>

### CODE REVIEW

**Review by pedrolamas — DISMISSED** [review]
LGTM

**Review by dweight — DISMISSED** [review]

**Comment by plloppii** [comment]
This is well needed! Woohoo!!!!

File: docs/development/index.md [hunk 1]
```diff
@@ -12,4 +12,38 @@ Fluidd is built using VueJS, and the Vuetify Framework to provide a cohesive,
...
+Follow the instructions from [NodeSource](https://github.com/nodesource/distributions/blob/master/README.md#deb)
+under "Node.js v14.x". For example, on Ubuntu flavored Linux distributions,
+run:
+
+```bash
+# Using Ubuntu
+$ curl -fsSL https://deb.nodesource.com/setup_14.x | sudo -E bash -
+$ sudo apt-get install -y nodejs
+```
+
```
Comment (plloppii) [inline docs/development/index.md:28]: I would add a note to check that nodejs was installed properly by doing a ```nodejs --version``` and a ```npm --version```

Comment (pedrolamas) [reply docs/development/index.md:28]: That's a very good point and always a nice thing to check!

Comment (dweight) [reply docs/development/index.md:28]: @matthewlloyd Would you mind adding it into your changes?

File: docs/development/index.md [hunk 2]
```diff
@@ -12,4 +12,47 @@ Fluidd is built using VueJS, and the Vuetify Framework to provide a cohesive,
 easily to implement UI.
 {: .fs-6 .fw-300 }
 
-Further details on development environments will come at a later date.
+## Running Fluidd locally
+
+### Install Node v14
+
+Follow the instructions from [NodeSource](https://github.com/nodesource/distributions/blob/master/README.md#deb)
```
Comment (ForsakenHarmony) [inline docs/development/index.md:19]: I don't think recommending some third-party installer is the best
https://nodejs.org/en/ is the official one

**Review by ForsakenHarmony — CHANGES_REQUESTED** [review]

File: docs/development/index.md (hunk 2 above)
Comment (matthewlloyd) [reply docs/development/index.md:19]: NodeSource is the official source of binary distributions for Node.js installed on Linux distros via a package manager:

https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions

Comment (ForsakenHarmony) [reply docs/development/index.md:19]: Ah intersting, but it's for "Debian and Ubuntu based Linux distributions"

So I'd still prefer linking to the official site, not everyone uses debian/ubuntu or even linux

**Review by dweight — DISMISSED** [review]

**Review by pedrolamas — APPROVED** [review]

**Pull request merged**